src/visualizations/config/adapters/dhis_highcharts/xAxis/scatter.js

```diff
@@ -12,20 +12,26 @@ import { getAxis } from '../../../../util/axes'
 import {
     getGridLineColor,
     getLabels,
-    getMaxValue,
     getMinValue,
     getRegressionLine,
 } from '../axis'
+import { PROP_EXTREME_LINES } from '../../../../../modules/outliers'
 
 const AXIS_TYPE = 'RANGE'
 const AXIS_INDEX = 1
 
-export default function (layout, series) {
+export default function (layout, series, extraOptions) {
     const dataValues = series?.map(item => item.data).flat()
     const axis = getAxis(layout.axes, AXIS_TYPE, AXIS_INDEX)
+    const extremeObj = extraOptions.outlierHelper?.vars?.config?.extremeLines
+        ?.enabled
+        ? extraOptions.outlierHelper[PROP_EXTREME_LINES][0]
+        : null
+    const xMax = extraOptions.outlierHelper?.vars?.xyStats?.xMax
+
     return objectClean({
         min: getMinValue(axis.minValue, dataValues),
```

Comment:
Should `min` be taken from `extremeObj` too?

Reply:
Hmm no, that value will always be larger than the min data value.